q2_fondue/sequences.py

```diff
@@ -163,24 +163,32 @@ def _run_fasterq_dump_for_all(
 
 
 def _process_one_sequence(filename, output_dir):
-    """Renames sequence files to follow the required naming convention."""
+    """Renames sequence files to follow the required naming convention
+    and moves them to single/paired subfolders."""
+    # Renaming
     new_name, is_paired = None, False
     if filename.endswith('_1.fastq'):
         # paired-end _1
-        acc = re.search(r'(.*)_1\.fastq$', filename).group(1)
+        acc = re.search(r'^.*\/(.*)_1\.fastq$', filename).group(1)
         new_name, is_paired = '%s_00_L001_R1_001.fastq' % acc, True
     elif filename.endswith('_2.fastq'):
         # paired-end _2
-        acc = re.search(r'(.*)_2\.fastq$', filename).group(1)
+        acc = re.search(r'^.*\/(.*)_2\.fastq$', filename).group(1)
         new_name, is_paired = '%s_00_L001_R2_001.fastq' % acc, True
     elif filename.endswith('.fastq'):
         # single-reads
-        acc = re.search(r'(.*)\.fastq$', filename).group(1)
+        acc = re.search(r'^.*\/(.*)\.fastq$', filename).group(1)
         new_name, is_paired = '%s_00_L001_R1_001.fastq' % acc, False
```

Comment:
This could simply become something like `_01_L001_R1_001.fastq` to distinguish from `_00_L001...` for paired reads.

Reply:
I agree with you that renaming would be a simpler solution. I have not implemented it this way as I had assumed that the number of the barcode has a particular meaning in the Casava naming format. If this is not the case, differentiating between the two with the barcode sounds like the better solution 😅.